common/proxy/proxy.go

```diff
@@ -151,6 +151,10 @@ func (p *Proxy) Implement(v common.RPCService) {
 				for k, value := range m {
 					inv.SetAttachments(k, value)
 				}
+			} else if m2, ok2 := atm.(map[string]interface{}); ok2 {
```

Comment:
we'd better leave some comments here so that others can understand the context better.

Reply:
done

Reply:
done

Reply:
IMO, the comment should be aligned with the else branch in the same line or under the else branch if it is too long.

Reply:
done